packages/kolibri-tools/lib/production.js

```diff
@@ -16,13 +16,18 @@ function webpackConfig(pluginData) {
       minimize: true,
       debug: false,
     }),
+    // requires >= v3.0.0, which is specified in the kolibri-core yarn workspace
+    new webpack.NormalModuleReplacementPlugin(
+      /^vue-intl$/,
+      'vue-intl/dist/vue-intl.prod.min.js'
+    ),
```

Comment:
As noted in the comment - it's a little weird that I reference `vue-intl` in `kolibri-tools`, but am relying on the version which is pinned in in `kolibri-core`. Do you think this is problematic?